shell/renderer/electron_renderer_client.cc

```diff
@@ -217,19 +217,17 @@ void ElectronRendererClient::SetupMainWorldOverrides(
   if (!prefs.webview_tag)
     return;
   // Setup window overrides in the main world context
-  // Wrap the bundle into a function that receives the isolatedWorld as
+  // Wrap the bundle into a function that receives the nodeProcess as
```

Comment:
Isn't leaking `nodeProcess` like this still violating the rules of context isolation, we need to remove all the arguments from this call.

Reply:
the comment should have been "... receives the isolatedWorld and nodeProcess ..." before.
I need process to access the v8Utils and webFrame bindings, I will check if it would be possible to get rid of them

Reply:
`nodeProcess` is now removed